mypyc/test-data/irbuild-basic.test

```diff
@@ -1840,7 +1840,8 @@ def h():
     r2, r3 :: object
     r4, r5 :: dict
     r6 :: str
-    r7, r8 :: object
+    r7 :: object
+    r8 :: object
```

Comment:
I have located the source cause of this problem. I'll submit a separate PR for that since it's going to change the IR tests in a lot of places(originally when we introduce const_int_regs into pretty IR printing, we didn't find out this happening)